scripts/apply_trilinos_patches.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+hashes=('545269e1838282d98c4a113e17e6322d9c87fe8b')
+
+cd ${TRILINOS_DIR}
+for i in "${hashes[@]}"
+do
+  wget https://github.com/trilinos/trilinos/commit/${i}.patch
+  git apply --whitespace=nowarn ${i}.patch
```

Comment:
But you still need to check whether it failed or not, right? Or are you saying that when it dies it does not set the return code?